include/aikido/control.hpp

```diff
@@ -1,13 +1,19 @@
+#include "aikido/control/EffortExecutor.hpp"
```

Comment:
I think some of these may no longer exist?

Reply:
Fixed, thanks!